src/arm/BBORG_SERVO-00A2.dts

```diff
@@ -75,45 +75,27 @@
     #address-cells = <1>;
     #size-cells = <0>;
 
-    pca9685_servo: pca9685_serve@7f {
-        compatible = "nxp, pca9685";
+    pca: pca@70 {
+        compatible = "nxp,pca9685-pwm";
+        #pwm-cells = <16>;
+        reg = <0x70>;
         lable = "pca9685_servo";
```

Comment:
Is this the correct spelling for 'lable'?

Reply:
no, it should be `label` 😂